script/States/App/Tests/WorkerTest.lua

```diff
@@ -3,23 +3,44 @@ local WorkerTest = require('States.Application')
 function WorkerTest:onInit()
     Log.Info("WorkerTest:onInit: Start")
 
-    Worker.AddWorkers({ "TestWorker" })
-    if TaskQueue:startWorker(Worker.TestWorker, "TestWorker", "script/States/App/Tests/TestWorkerFunction.lua") == false then
-        Log.Error("Cannot start worker")
-    end
-    local expectedTaskId = TaskQueue:sendTask(Worker.TestWorker, "TestPayload")
-    if expectedTaskId == nil then
-        Log.Error("Cannot send task")
-    end
+    WorkerId.AddWorkers({ "TestWorker", "TestWorker2" })
```

Comment:
What do you think of `LuaWorkers` instead of `WorkerId`

Reply:
WorkerId is an enum in the engine that contains ids of the engine workers, and it can be extended on the Lua side. So it's not really worker but just a type/id

Reply:
Personally, I think `WorkerId` is fine. The way I see this is that we're adding new worker types to the `WorkerId` enum. Adding "Lua" feels redundant here, beacuse we're already in Lua. However, I do think "AddWorkers" is a bit redundant as we know they're workers due to the type name, so perhaps rename `AddWorkers` to `Register` i.e. `WorkerId.Register({ "TestWorker", "TestWorker2" })`

Reply:
Isn´t it kinda weird that registering to an enum will *actually* in the background create a new accessible lua state / worker. Feels a bit off to me.